travis_osx_steps.sh

```diff
@@ -4,8 +4,9 @@ set -e
 
 if [ "$PLAT" == "arm64" ] || [ "$PLAT" == "universal2" ]; then
   if [[ "$(xcrun -show-sdk-version)" == 10.*  ]]; then
-    if ([ "$GITHUB_WORKFLOW" != "" ] || [ "$PIPELINE_WORKSPACE" != "" ]) && [ $(ls /Applications | grep Xcode_12.*\.app) ]; then
-      sudo xcode-select -switch /Applications/Xcode_12.*.app
+    latestXcode=$(ls /Applications | grep Xcode[_0-9\.]*\.app | sort -V | tail -n 1)
```

Comment:
And is that true - that we always want the latest?

Reply:
This code only runs in the event that `xcrun -show-sdk-version` is 10.* - meaning that a later version is required than what is currently selected. At the present time, the highest version of Xcode can compile for arm64. One would expect that to continue to be the case for several years to come.

If a different version of Xcode is required, I would have thought choosing the latest one is best, and what most users would prefer?

Reply:
That depends if we later run into a situation where later Xcode generates binaries that are not compatible with older arm64s.   I guess that's very unlikely in the near future.